### ISSUE TITLE
Messages with ASCII smileys are now annoying to type

### ISSUE THREAD

**Issue by bertjwregeer** [keybase/client#15595]
I tend to use smileys like `:-)` and `:-P` often while typing. With the new `:smileys:` that is no longer possible because `:-)` gets turned into the first suggestion which is `:flag-cc:`

<img width="595" alt="image" src="https://user-images.githubusercontent.com/649426/51370478-80de5280-1ab4-11e9-8b9e-c1cff6f3baa9.png">

**Comment by mmerickel** [comment]
Yeah it's especially annoying because I tend to do it at the end of the message so my next key is <return> and it's hard to stop hitting that button to escape the menu.

**Comment by svoop** [comment]
This seems to no longer be a problem and IMO can be closed.
